dev/io.openliberty.transport.http_fat/bnd.bnd

```diff
@@ -0,0 +1,40 @@
+#*******************************************************************************
+# Copyright (c) 2018, 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
+	test-applications/app1/src
+	
+fat.project: true
+
+# Define additional tested features that are NOT present in any XML files in this bucket.
+# In this case, Java EE 8 features are added programmatically at runtime by the RepeatTests rule.
+tested.features:\
+	servlet-4.0, \
+	servlet-5.0
+
+-buildpath:\
+	io.openliberty.org.apache.commons.logging;version=latest, \
+	io.openliberty.org.apache.commons.codec;version=latest, \
+	com.ibm.ws.http2.client;version=latest, \
```

Comment:
can you remove this `com.ibm.ws.http2.client`?

Reply:
Yes.